### PR TITLE
add depth compare example

### PR DESCRIPTION
**Pull request by pablovela5620**
### What
A new example for Monocular Depth estimation.

This example demonstrates using rerun to compare different monocular depth estimation networks including [DepthAnythingV1/V2](https://github.com/LiheYoung/Depth-Anything), [Unidepth](https://github.com/lpiccinelli-eth/UniDepth/tree/main), and [Metric3DV2](https://github.com/YvanYin/Metric3D). It allows easy loading of each network and runs them side by side to show the depth map, disparity map, back project to camera space, and point cloud. This includes a huggingface space and an external repository. This is currently a draft PR as I'm waiting for [gradio rerun viewer to upgrade to 0.17](https://github.com/radames/gradio-rerun-viewer/pull/3). This is to make it easy to use a viewer per tab, only with setting the image plane distance with blueprint
[Huggingface Space](https://huggingface.co/spaces/pablovela5620/depth-compare)
DepthCompare

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/6885?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/6885?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!
* [x] If have noted any breaking changes to the log API in `CHANGELOG.md` and the migration guide

- [PR Build Summary](https://build.rerun.io/pr/6885)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

File: examples/python/depth_compare/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!--[metadata]
+title = "Depth Compare"
+tags = ["2D", "3D", "HuggingFace", "Depth", "Pinhole camera"]
+source = "https://github.com/pablovela5620/monoprior"
+thumbnail = "TODO"
+thumbnail_dimensions = [480, 480]
```
Comment (02alexander) [inline examples/python/depth_compare/README.md:6]: Thumbnail needs to be added.

**Comment by 02alexander** [comment]
Looks good overall. The example must be added to the `examples/manifest.toml` for it to show on the rerun.io/examples page.

**Review by 02alexander — CHANGES_REQUESTED** [review]

**Comment by pablovela5620** [comment]
Add depth_compare to the generative-vision section and also added the thumbnail!

File: examples/python/depth_compare/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<!--[metadata]
+title = "Depth Compare"
```
Comment (02alexander) [inline examples/python/depth_compare/README.md:2]:
```suggestion
title = "Depth compare"
```

Comment (02alexander) [reply examples/python/depth_compare/README.md:2]: With problem from the linter

File: examples/python/depth_compare/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<!--[metadata]
+title = "Depth compare"
+tags = ["2D", "3D", "HuggingFace", "Depth", "Pinhole camera"]
+source = "https://github.com/pablovela5620/monoprior"
+thumbnail = "https://static.rerun.io/depth-compare-thumbnail/270fdb11a2b6e85c69d83d52d475a5ec644de9df/480w.png"
+thumbnail_dimensions = [480, 480]
```
Comment (02alexander) [inline examples/python/depth_compare/README.md:6]:
```suggestion
thumbnail_dimensions = [480, 258]
```

Comment (02alexander) [reply examples/python/depth_compare/README.md:6]: Fix another problem presented by the linter.

**Review by 02alexander — APPROVED** [review]

**Pull request merged**